quantized Conv1d

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #35134 Minor fix to quantized conv docstring
* **#35093 quantized Conv1d**

Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)

Commit:
quantized Conv1d

Commit:
Update on "quantized Conv1d"

Commit:
Update on "quantized Conv1d"


Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)

Commit:
Update on "quantized Conv1d"


Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)

Commit:
Update on "quantized Conv1d"


Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)

dskhudia commented on torch/nn/quantized/modules/conv.py:
```diff
@@ -136,6 +136,129 @@ def __setstate__(self, state):
+        >>> input = torch.randn(20, 16, 100)
+        >>> # quantize input to qint8
+        >>> q_input = torch.quantize_per_tensor(input, scale=1.0, zero_point=0, dtype=torch.qint32)
+        >>> output = m(input)
```
dtype = torch.quint8 and output=m(q_input)?

Commit:
Update on "quantized Conv1d"


Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)

Commit:
Update on "quantized Conv1d"


Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)

Commit:
Update on "quantized Conv1d"


Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)

Commit:
Update on "quantized Conv1d"


Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)

Commit:
Update on "quantized Conv1d"


Differential Revision: [D20555070](https://our.internmc.facebook.com/intern/diff/D20555070)
